src/app/code/community/Omise/Gateway/Controller/Base.php

```diff
@@ -1,19 +1,194 @@
 <?php
 abstract class Omise_Gateway_Controller_Base extends Mage_Core_Controller_Front_Action
 {
+    /**
+     * @var string
+     */
+    protected $title;
```

Comment:
Unless we need to change this (and many others below) at any point, I think we'd be better off just having them as class constants (or maybe public variables). The getters and setters certainly aren't needed - they are just adding unnecessary overhead